packet/cloud.go

```diff
@@ -73,17 +74,22 @@ type Config struct {
 	AnnotationLocalASN   string `json:"annotationLocalASN,omitEmpty"`
 	AnnotationPeerASNs   string `json:"annotationPeerASNs,omitEmpty"`
 	AnnotationPeerIPs    string `json:"annotationPeerIPs,omitEmpty"`
+	EIPTag               string `json:"eipTag,omitEmpty"`
 }
 
 func newCloud(packetConfig Config, client *packngo.Client) (cloudprovider.Interface, error) {
-	return &cloud{
+	c := &cloud{
 		client:       client,
 		facility:     packetConfig.Facility,
 		instances:    newInstances(client, packetConfig.ProjectID),
 		zones:        newZones(client, packetConfig.ProjectID),
 		loadBalancer: newLoadBalancers(client, packetConfig.ProjectID, packetConfig.Facility, packetConfig.DisableLoadBalancer, packetConfig.LoadBalancerManifest, packetConfig.LocalASN, packetConfig.PeerASN),
 		bgp:          newBGP(client, packetConfig.ProjectID, packetConfig.LocalASN, packetConfig.PeerASN, packetConfig.AnnotationLocalASN, packetConfig.AnnotationPeerASNs, packetConfig.AnnotationPeerIPs),
-	}, nil
+	}
+	if packetConfig.EIPTag != "" {
+		c.controlPlaneEndpointManager = newControlPlaneEndpointManager(packetConfig.EIPTag, packetConfig.ProjectID, client.DeviceIPs, client.ProjectIPs, c.instances)
+	}
+	return c, nil
```

Comment:
This here kind of breaks the style of the others. Is there a reason not to do it the same as the others

```go
return &cloud{
 		client:       client,
 		facility:     packetConfig.Facility,
 		instances:    newInstances(client, packetConfig.ProjectID),
 		zones:        newZones(client, packetConfig.ProjectID),
 		loadBalancer: newLoadBalancers(client, packetConfig.ProjectID, packetConfig.Facility, packetConfig.DisableLoadBalancer, packetConfig.LoadBalancerManifest, packetConfig.LocalASN, packetConfig.PeerASN),
 		bgp:          newBGP(client, packetConfig.ProjectID, packetConfig.LocalASN, packetConfig.PeerASN, packetConfig.AnnotationLocalASN, packetConfig.AnnotationPeerASNs, packetConfig.AnnotationPeerIPs),
                controlPlaneEndpointManager: newControlPlaneEndpointManager(packetConfig.EIPTag, packetConfig.ProjectID, client.DeviceIPs, client.ProjectIPs, c.instances),
 	}, nil
```

and then have `controlPlaneEndpointManager.nodeReconciler()` return nil if the first arg to `newControlPlaneEndpointManager()` is `""`?